examples/etcd/druid_v1alpha1_etcd.yaml

```
@@ -54,7 +54,7 @@ spec:
     #   provider: local # options: aws,azure,gcp,openstack,alicloud,dell,openshift,local
     #   prefix: etcd-test
     compression:
-      enabled: false
+      enabled: true
```

Since omitting the `compression` block would mean compression is enabled, perhaps we can show in the example how to disable it instead?
The current proposed changes set it to `true`, which doesn't really change the behavior from the default.

```suggestion
      enabled: false # compression is enabled by default, and can be disabled like so.
```

---

As discussed, we would like to have enabled set to `true`, so that user by mistake should disable the compression while do testing.